.github/workflows/dapr-deploy.yml

```
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/master/install/install.sh
-      DAPR_RUNTIME_VER: 1.14.0-rc.7
+      DAPR_RUNTIME_VER: 1.14.0-rc.8
       DAPR_CLI_VER: 1.14.0-rc.6
```

```suggestion
       DAPR_CLI_VER: 1.14.0-rc.8
```
Minor nit but I think we should bump the cli version too